.werft/installer-tests.ts

```diff
@@ -154,8 +185,8 @@ export async function installerTests(config: TestConfig) {
     }
 }
 
-function callMakeTargets(phase: string, description: string, makeTarget: string) {
-    werft.phase(phase, description);
+function callMakeTargets(phase: string, description: string, makeTarget: string, args: string[]) {
```

Comment:
My understanding is that `args` are somehow passed to the `make` invocation, but I'm not seeing that in this code. Am I missing something?

Reply:
I have removed this `args` usage. You are right, ultimately super hacky!